seqr/management/tests/load_rna_seq_tpm_tests.py

```diff
@@ -77,21 +73,26 @@ def test_command(self, mock_open, mock_logger, mock_utils_logger, mock_gzip_open
         self.assertEqual(new_sample.tissue_type, 'WB')
 
         models = RnaSeqTpm.objects.all()
-        self.assertEqual(models.count(), 4)
-        self.assertSetEqual({model.sample for model in models}, {existing_sample, existing_sample2, new_sample})
-        self.assertEqual(models.get(sample=existing_sample, gene_id='ENSG00000240361').tpm, 12.6)
+        self.assertEqual(models.count(), 5)
+        self.assertSetEqual({model.sample for model in models}, set(existing_rna_samples))
+        self.assertEqual(len(models.filter(sample=existing_sample, gene_id='ENSG00000240361')), 0)
```

Comment:
nitpicky, but `models.filter().count()` is a bit better than `len(model.filters())`. It mean django executes the more efficient COUNT query on the database and never actually pulls out the records